components/multimedia/soundfont/timgm6mb/Makefile

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Do we really have to place it in 'components/multimedia/soundfont/timegm6mb'? I'd put it one level upper (avoid creating soundfont directory to hold one component).